pkg/operator/controllers/muo/deploy.go

```diff
@@ -53,7 +72,46 @@ func (o *deployer) Resources(pullspec string) ([]kruntime.Object, error) {
 		// set the image for the deployments
 		if d, ok := obj.(*appsv1.Deployment); ok {
 			for i := range d.Spec.Template.Spec.Containers {
-				d.Spec.Template.Spec.Containers[i].Image = pullspec
+				d.Spec.Template.Spec.Containers[i].Image = config.Pullspec
+			}
+		}
+
+		if cm, ok := obj.(*corev1.ConfigMap); ok {
```

Comment:
The configmap can be edited, how can the defaults be restored?

Reply:
the configmap is marked as owned by the MUO controller, so it will cause a reconcile and rewrite